k8s/src/main/scala/io/buoyant/k8s/Api.scala

```diff
@@ -0,0 +1,187 @@
+package io.buoyant.k8s
```

Comment:
thoughts about scoping this under a v1 package?  Since the API is versioned, and all of this is (i think?) specific to the v1 api, should we try to reflect that in the packaging?